modules/uncertainty/R/ensemble.R

```diff
@@ -200,35 +200,43 @@ get.ensemble.samples <- function(ensemble.size, pft.samples, env.samples,
 write.ensemble.configs <- function(defaults, ensemble.samples, settings, model, 
                                    clean = FALSE, write.to.db = TRUE,restart=NULL) {
   
+  con <- NULL
```

Comment:
This seems unnecessary. You're just redefining this variable below.

Reply:
@ashiklom if we just define the `con` in the scope of the `if` then it would break all the ifs afterwards.

Reply:
Ah, OK, I see that now.